tasks.py.jinja

```diff
@@ -83,7 +101,7 @@ def write_code_workspace_file(c, cw_path=None):
                         "remoteRoot": "/opt/odoo",
                     }
                 ],
-                "port": 12899,
+                "port": int(ODOO_VERSION)*1000 + 899,
```

Comment:
This file has now become a `.py.jinja` file, so Black will not format it here in the template. We have to apply pretty-formatting by hand, just like in the old times:

```suggestion
                "port": int(ODOO_VERSION) * 1000 + 899,
```